examples/ping.go

```diff
@@ -0,0 +1,21 @@
+package main
```

Comment:
Example name "ping" is confusing because everyone is used to the "ping" command which pings some other host. This example replies to pings and doesn't send its own ping requests. BTW a ping command implementation would be a very good example.